src/doc/unstable-book/src/compiler-flags/sanitizer.md

```diff
@@ -164,17 +164,25 @@ Shadow byte legend (one shadow byte represents 8 application bytes):
   Left alloca redzone:     ca
   Right alloca redzone:    cb
   Shadow gap:              cc
-==424427==ABORTING
+==39249==ABORTING
 ```
 
-## MemorySanitizer
+# MemorySanitizer
+
+MemorySanitizer is detector of uninitialized reads. It is supported only on
+`x86_64-unknown-linux-gnu` target.
+
+MemorySanitizer requires all program code to be instrumented. Failing to achieve
+that will result in false positive reports.
```

Comment:
This means "if you link to C code it needs to be compiled with memorysantizer too", right?

Reply:
Yes, expanded on this a bit to give more context to the example that follows.

In general this is a major challenge when using MemorySanitizer in larger code
bases. For those written mostly in Rust and using Cargo, it works surprisingly
well.